src/main/java/org/killbill/billing/plugin/gocardless/GoCardlessPaymentPluginApi.java

```diff
@@ -57,15 +57,15 @@
 import org.killbill.billing.ObjectType;
 
 public class GoCardlessPaymentPluginApi implements PaymentPluginApi {
-
 	private static final Logger logger = LoggerFactory.getLogger(GoCardlessPaymentPluginApi.class);
-	private OSGIKillbillAPI killbillAPI;
-	private Clock clock;
-	private static String GC_ACCESS_TOKEN_PROPERTY = "GC_ACCESS_TOKEN";
-
-	private GoCardlessClient client;
+	private static final String GC_ACCESS_TOKEN_PROPERTY = "GC_ACCESS_TOKEN";
```

Comment:
Gateway tokens are typically being configured using per-tenant plugin properties (and not environment variables). For example, stripe plugin will extract this [config](https://github.com/killbill/killbill-stripe-plugin/blob/master/src/main/java/org/killbill/billing/plugin/stripe/StripePaymentPluginApi.java#L122) from the per-tenant defined properties.

Was there a reason to do it this way?